try:
    if(nombre>3):       #Esto dará un error porque nombre no está definido, entonces saltará al except
        print("Hola")
except:
    print("Algo salió mal con el código, por favor revisalo")

